client/login/wp-login/login-links.jsx

```diff
@@ -247,6 +247,31 @@ export class LoginLinks extends Component {
 		);
 	}
 
+	renderQrCodeLoginLink() {
+		if ( ! config.isEnabled( 'login/qr-code-login' ) || this.props.twoFactorAuthType ) {
+			return null;
+		}
+		if ( this.props.isLoggedIn ) {
+			return null;
+		}
+
+		// Is not supported for any oauth 2 client.
+		if ( this.props.oauth2Client ) {
+			return null;
+		}
+
+		if ( this.props.isJetpackWooCommerceFlow ) {
+			return null;
+		}
+
+		const loginUrl = login( {
+			locale: this.props.locale,
+			twoFactorAuthType: 'qr',
+			signupUrl: this.props.query?.signup_url,
+		} );
+		return <a href={ loginUrl }>{ this.props.translate( 'Login via the Mobile App' ) }</a>;
```

Comment:
Sentence case here too:

```suggestion
		return <a href={ loginUrl }>{ this.props.translate( 'Login via the mobile app' ) }</a>;
```